pandas/tests/indexes/test_datetimelike.py

```diff
@@ -721,6 +721,17 @@ def test_fillna_datetime64(self):
                            dtype=object)
             self.assert_index_equal(idx.fillna('x'), exp)
 
+    def test_contains(self):
+        #GH13572
+        dates = ['2015-01-03', '2015-01-01', '2015-01-04', '2015-01-05', '2015-01-02']
+        monotonic = pd.to_datetime(sorted(dates))
+        non_monotonic = pd.to_datetime(['2015-01-03', '2015-01-01', '2015-01-04', '2015-01-05', '2015-01-02'])
```

Comment:
How is NaT supposed to work with period indices?
The only NaT-like object I can successfully test with is this one
```python
>>> pd.Period('NaT', freq='D') in pd.period_range('2015-01-01', periods=5, freq='D').insert(0, pd.NaT)
True
```
Any of `pd.NaT, None, float('nan'), np.nan` produce `False` as result, although they work on Datetimeindex objects with NaT's in them.